### PR TITLE
Fix Dropdown menus not closing when clicking on iframe (space pages)

### PR DESCRIPTION
**Pull request by beurkinger**

### CODE REVIEW

File: js/src/lib/components/DropdownMenu/DropdownMenu.svelte [hunk 1]
```diff
@@ -26,10 +27,12 @@
 
 		return () => {
 			document.removeEventListener("click", handleClickDocument);
+			window.removeEventListener("blur", onClose);
 		};
 	});
 
 	function handleClickDocument(e: MouseEvent) {
+		console.log("CLICK");
```
Comment (mishig25) [inline js/src/lib/components/DropdownMenu/DropdownMenu.svelte:35]:
```suggestion
```

Comment (beurkinger) [reply js/src/lib/components/DropdownMenu/DropdownMenu.svelte:35]: 🙈

Comment (osanseviero) [reply js/src/lib/components/DropdownMenu/DropdownMenu.svelte]: nobody saw that

**Review by osanseviero — APPROVED** [review]
Nice :smile:

**Pull request closed without merging**